controllers/apps/cluster_controller.go

```diff
@@ -107,8 +107,8 @@ import (
 // +kubebuilder:rbac:groups=core,resources=serviceaccounts,verbs=get;list;watch
 // +kubebuilder:rbac:groups=core,resources=serviceaccounts/status,verbs=get
 
-// +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=rolebindings,verbs=get;list;watch
```

Comment:
this looks like hack to me, do aware that we are already employing `rolebinding` mechanism, adding is acceptable but not refactor it out.

Reply:
@xuriwuyun do review this PR

Reply:
reverted